src/core/qgsdistancearea.h

```diff
@@ -222,6 +222,22 @@ class CORE_EXPORT QgsDistanceArea
      */
     double measureLine( const QgsPoint &p1, const QgsPoint &p2, QgsUnitTypes::DistanceUnit &units ) const;
 
+    /**
+     * Calculates distance from one point with distance in meters and azimuth (direction)
+     * When the sourceCrs is Geographic, computeSpheroidProject will be called
+     * otherwise QgsPoint.project() will be called after QgsUnitTypes::fromUnitToUnitFactor() has been applied to the distance
+     * \note:
+     *  The input Point must be in the CoordinateReferenceSystem being used
+     * \since QGIS 3.0
+     * \param p1 start point [can be Cartesian or Geographic]
+     * \param distance must be in meters
+     * \param projected_point calculated projected point
```

Comment:
Should be `\param projectedPoint` (that's causing the build warnings which have failed on Travis)

You'll also need to add `\param azimuth` as it's missing and causing a build warning too